tools/lint/lint.py

```diff
@@ -957,6 +957,8 @@ def create_parser():
                         "working directory, not just files that changed")
     parser.add_argument("--github-checks-text-file", type=ensure_text,
                         help="Path to GitHub checks output file for Taskcluster runs")
+    parser.add_argument("-j", "--jobs", type=int, default=0,
```

Comment:
Presumably it's possible to pass in e.g. -1 as a value here.

In general I think I'd prefer to use `None` to represent the missing value and range check the argument. But that's not a blocker if you'd prefer not to change things.

Reply:
There's a number of existing tools that take `0` as a value that auto-detects. Certainly we can make this default to `None` and override that with `0` ourselves, but that seems silly…

Reply:
"that's not a blocker if you'd prefer not to change things"

I just don't really like this kind of API which used magic numbers because at some point someone wrote some C code where the variable had type `int` and didn't have the option to do anything other than stuff additional meaning into out-of-range values.

Reply:
IMO there should be some way to access the default while still passing `--jobs`, and I'd much rather we matched what other software does rather than introduce a new way.